client/src/components/Comment/index.tsx

```diff
@@ -1,20 +1,29 @@
+/* eslint-disable no-unreachable */
+/* eslint-disable no-undef */
 import './style.css'
 import { FC } from 'react'
-import profile from '../../assets/images/profile.png'
-import commentPic from '../../assets/images/commentPic.png'
+import ICommentsData from '../../interfaces/props/CommentsProps'
 
-const Comment:FC = () => (
-  <div className="comment">
-    <div className="user-info">
-      <img src={profile} alt="profile-img" />
-      <div>
-        <p className="username">saif hayek</p>
-        <p className="date">2022-09-10 at 09:57:55</p>
+const Comment:FC<ICommentsData> = ({
+  id, EventId, image, content, createdAt, User,
+}):JSX.Element => {
+  const dateFormat = new Date(createdAt)
+  const createdAtFormatted = `${dateFormat.getHours()}
+  :${dateFormat.getMinutes()},${dateFormat.toDateString()}`
```

Comment:
you can use `dayjs` or `moment.js` for date formatting
or you can add this functionality to our helpers folder
because i have noticed you used it in other place